parsing/dialects/intermediate/dialects/dialect1/parserImpls.ftl

```diff
@@ -4695,3 +4697,19 @@ SqlRenameProcedure SqlRenameProcedure() :
         return new SqlRenameProcedure(getPos(), oldProcedure, newProcedure);
     }
 }
+
+// Semicolon is optional after the last statement.
+SqlBeginRequestCall SqlBeginRequestCall() :
+{
+    final SqlStatementList statements = new SqlStatementList(getPos());
+    final Span s = Span.of();
+    SqlNode e;
+}
+{
+    <BEGIN> <REQUEST>
+    e = SqlStmt() { statements.add(e); }
```

Comment:
Can we use SqlStmtList() for this, which says that a SEMICOLON is optional after the last statement?

Reply:
It looks like it would parse correctly but it returns a different type of SqlNodeList instead of SqlStatementList. I think if we wanted to update that function it would require adding an import in every config file for SqlStatementList then updating the tests that use this function?